local-config/config.yaml

```diff
@@ -0,0 +1,10 @@
+http:
+  prefix: someprefix
+maxWaitTimeForTrace: 10s
+postgresConnString: "host=127.0.0.1 user=tracetest password=not-secure-database-password  port=5432 sslmode=disable"
+jaegerConnectionConfig:
+  endpoint: localhost:16685
```

Comment:
unless really neccesary, I'd try to use the docker internal network as much as possible. I imagine tracetest should be able to talk with jaeger using the `jaeger:16685` url